packages/sanity/tsdoc.json

```
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://developer.microsoft.com/json-schemas/tsdoc/v0/tsdoc.schema.json",
+    "tagDefinitions": [
+        {
+          "tagName": "@hidden",
+          "syntaxKind": "block",
+          "allowMultiple": true
+        }
+      ],
+    
+      "supportForTags": {
+        "@hidden": true,
+        "@beta": true,
+        "@internal": true,
+        "@public": true,
+        "@experimental": true,
+        "@see": true,
+        "@link": true,
+        "@example": true,
+        "@deprecated": true,
+        "@alpha": true
```

I believe this is missing `@param` and `@returns` tag causing eslint error

```suggestion
        "@alpha": true,
        "@param": true,
        "@return": true
```

---

made a typo in code suggestion 🤦 pushed a fix